src/store/auth.js

```diff
@@ -0,0 +1,63 @@
+import { apiClient } from '../modules/apiClient'
+import { storageKey } from '../modules/storageKey'
+
+export const AUTH_REQUEST = 'AUTH_REQUEST'
+export const AUTH_LOGOUT = 'AUTH_LOGOUT'
+export const AUTH_SUCCESS = 'AUTH_SUCCESS'
+export const AUTH_ERROR = 'AUTH_ERROR'
+
+export const USER_REQUEST = 'USER_REQUEST'
+
+export const authModule = {
```

Comment:
a czy tutaj nie trzeba dodać `namespaced: true,` ? zdaje mi się że jak dodać kolejne moduły to atrybuty o tych samych nazwach będą Ci się nadpisywać

Reply:
Bardzo możliwe, że masz racje, bo na razie nie ma żadnego namespace'u, chociaż chciałem, żeby był